### PR TITLE
Add match to docs

### LINKED ISSUE #18: switch or match
I Think we should also add some sort of a switch logic to the language.  
We could add oldschool switch logic here but i must say i really enjoy the rust match logic.  

Sadly the match statement is kinda special to rust as far as i can see but this does not have to be an issue because a match statement is simply said a more advanced `if else if else if ...`.  
If we would have support for parsing match statements we could add a helper function that converts a match to a `if else if ..` or switch if the match is simple.

### PR DESCRIPTION
**Pull request by TheOtterlord**
- Add match to **Todo**
- Add match to docs
- Closes #18

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,6 +41,7 @@ This roadmap is used to track the progress on the project. If you add a feature
           * [x]  `while true {}`
           * [x]  `for foo in bar {}`
           * [ ]  `if foo {} else if bar {} else {}`
+          * [ ]  `match foo {bar => baz}`
```
Comment (mjarkk) [inline README.md:44]: Maybe we can make these a bit more extensive so it's more welcoming to start working on just one of these items
```suggestion
          * [ ] if
            * [ ] `if foo {}`
            * [ ] `if foo {} else {}`
            * [ ] `if foo {} else if bar {} else {}`
          * [ ] match
            * [ ] `match foo { }`
            * [ ] `match foo { _ => {} }`
            * [ ] `match foo { bar => {} _ => {} }`
```

File: docs/Conditions.md [hunk 2]
```diff
@@ -44,3 +45,16 @@ if condition1 {
...
+
+Another way of comparing multiple conditions is to use the `match` statement.
+
+```rust
+match foo {
+  // if foo == bar
+  bar => do_something()
+  // else if foo == baz
+  baz => do_something_else()
+}
```
Comment (mjarkk) [inline docs/Conditions.md:59]: Maybe also add a default case:  
```rust
  _ => do_default()
```

Comment (TheOtterlord) [reply docs/Conditions.md:61]: Done :-)

**Comment by mjarkk** [comment]
Great lets merge this

**Pull request merged**